clientconn.go

```diff
@@ -75,6 +75,7 @@ type dialOptions struct {
 	codec    Codec
 	cp       Compressor
 	dc       Decompressor
+	strategy backoffStrategy
```

Comment:
@iamqizhao I had `backoff` originally but it stutters. Most uses would be `cc.dopts.backoff.backoff(n)`. I'll resubmit with this as `bs` but I'm open to other suggestions.